src/vstart.sh

```diff
@@ -189,9 +189,12 @@ inc_osd_num=0
 
 msgr="21"
 
+rotate_logs=1
```

Comment:
I suggest to keep the legacy behaviour as default: `rotate_logs=0` and `--rotate-logs` instead of `--dont-rotate`.

Reply:
For vstart.sh too? vstart_runner doesn't rotate logs by default. See https://gist.github.com/rishabh-d-dave/b5871cf526e5da0e8551cc0687ecccb5#file-without-rotate-logs.

Reply:
Yup, but most vstart.sh users don't even use vstart_runner at all, so I think it's better to keep the legacy behaviour by default. 

As a rule of thumb, when adding new behaviours to software, my approach is either I ensure it has negligible impact on people, or then I spend some time educating them on the implications of that change. As software is always easier to 'fix' than people, I personally lean towards the first option. Additionally, a change is always better received when it's seen as optional...

Reply:
@epuertat For vstart.sh, this code is just setting up a configuration file for logrotate. I've just had a discussion with @rishabh-d-dave about this. I've suggested that we actually remove this vstart.sh switch (`--dont-rotate`) entirely because there's really no reason **not** to write out the logrotate config to the build directory. Nowhere in `vstart.sh` is logrotate actually run.

Reply:
@epuertat The intention for just setting up the configuration file for logrotate is that the vstart users can easily run logrotate themselves when required.

Reply:
Ah, ok, thanks for the clarification @batrick !